Contexts.js

```diff
@@ -0,0 +1,3 @@
+import { createContext } from "react";
```

Comment:
I think I did the context file layout a little different in my login PR, but can address the potential merge conflict in future PR